templates/allauth/account/email.html

```diff
@@ -0,0 +1,73 @@
+{% extends "account/base.html" %}
```

Comment:
I would look at the indentation of this and maybe some of the the other templates.

Reply:
I'll go through them and format them appropriately. As above comment.